helm/application-namespace-prep/values.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "properties": {
+        "authnK8s": {
+            "required": [
+                "goldenConfigMap",
+                "namespace"
+            ],
+            "properties": {
+                "goldenConfigMap": {
+                    "type": "string",
+                    "maxLength": 252,
+                    "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
```

Comment:
The default error messages that Helm schema validation provides for pattern mismatches are pretty vague.
I think it would be REALLY helpful to add a more meaningful error message e.g. by adding this code below this line:
```
                    "message": {
                        "pattern": "The authnK8s.ConfigMap value must begin with alpanumeric blah, blah, blah..."
                    }
```
Same for the other patterns in the schema.

Reply:
Apologies for the noise, please ignore this comment about adding `"message"` entries. You're absolutely correct, this isn't supported... not sure why I thought otherwise when I was experimenting with your change set.